src/MapzenResponse.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  MapzenResponse.swift
+//  ios-sdk
+//
+//  Created by Sarah Lensing on 3/21/17.
+//  Copyright © 2017 Mapzen. All rights reserved.
+//
+
+import Foundation
+import Pelias
+
+public class MapzenResponse : NSObject {
```

Comment:
This should probably be something more descriptive like `SearchResponse`. 

Also it's not practice in swift to prefix names with things like `MZ` or `Mapzen` because namespaces are implicit. So if there's a namespace conflict you would use `ModuleName.Class.Function()` instead of `Class.Function()`. So in general I would remove that prefix from these classes / structs / etc. This is something I failed to understand while building Pelias, so lets fix that here :) 

There are always exceptions however. For example the `MapzenManager` is a good one since it's a state manager for Mapzen services, but otherwise I think we can be a little more loose with naming.